model/qti/choice/Choice.php

```diff
@@ -88,8 +90,9 @@ public function isIdentifierAvailable($newIdentifier)
                 $collection = $relatedItem->getIdentifiedElements();
 
                 try {
-                    $uniqueChoice = $collection->getUnique($newIdentifier, 'oat\\taoQtiItem\\model\\qti\\choice\\Choice');
-                    $uniqueOutcome = $collection->getUnique($newIdentifier, 'oat\\taoQtiItem\\model\\qti\\OutcomeDeclaration');
+                    $uniqueChoice = $collection->getUnique($newIdentifier, Choice::class);
```

Comment:
Are you sure that `Choice::class` === `oat\\taoQtiItem\\model\\qti\\choice\\Choice` ? 
I would avoid any type of change like this to reduce chances for bugs.